### PR TITLE
NO TASK / refactor and several fixes to the editor

### LINKED ISSUE #173: [Editor] Inline toolbar does not show
In the post editor (v2), the inline toolbar (with bold, italic, insert link) never becomes visible. Selecting text does not help.

As a result, I am completely unable to format my text or insert links.

The problem presents itself in both Firefox and Chromium. I can see the inline toolbar in the browser’s dev console but it always has display none. The toolbar appears to be functional after I manually set it to display block and assign a position.

### PR DESCRIPTION
**Pull request by nynevi**
- fixed bg-image not being visible on firefox or chromium,
- fixed the case where you could add images (and other media) to your comments,
- refactored editor component to be more readable and maintainable,
- refactor consists of using EditorJS's core onReady function being used to initialize angular side of things and data: {blocks: Block[]} attribute to initialize the body of the editor so we don't do it manually
- also fixes #173 where you are unable to see inline toolbar

- [x] runtime tested manually
- [x] tests run successfully
- [x] ssr built successfully

### CODE REVIEW

File: honestcash-v2/src/styles.scss [hunk 1]
```diff
@@ -19,7 +19,10 @@ body {
   background: linear-gradient(
       rgba(0, 0, 0, 0.4),
       rgba(0, 0, 0, 0.4)
-  ), url('https://github.com/honest-cash/honestcash/blob/redesign/hc-6/public/img/artwork-login.jpg?raw=true') no-repeat 50% fixed / cover;
+  ), url('https://github.com/honest-cash/honestcash/blob/redesign/hc-6/public/img/artwork-login.jpg?raw=true');
```
Comment (adrianbarwicki) [inline honestcash-v2/src/styles.scss]: This needs to be inside the repository.

**Comment by adrianbarwicki** [comment]
Is the PR properly tested? We should include in the description a short info that the PR has been tested and that it has been confirmed that the affected functionality is still working.

File: honestcash-v2/src/app/modules/editor/components/editor/editor.component.ts [hunk 2]
```diff
@@ -138,67 +122,104 @@ export class EditorComponent implements OnInit, OnDestroy {
...
+
+  private onEditorReady() {
+    this.editorService.savePostLocally(this.story);
+    this.store.dispatch(new EditorLoad());
+    this.hasEditorInitialized = true;
+  }
+
+  private setupEditorScripts() {
+    this.editorScripts = [
+      this.scriptService.loadScript('https://cdn.jsdelivr.net/npm/@editorjs/paragraph@2.5.1/dist/bundle.min.js'),
```
Comment (adrianbarwicki) [inline honestcash-v2/src/app/modules/editor/components/editor/editor.component.ts]: These need to be inside the repo as well so we don't get outside dependencies.

Comment (nynevi) [reply honestcash-v2/src/app/modules/editor/components/editor/editor.component.ts]: added them locally the same way as simple-bitcoin-wallet

**Comment by nynevi** [comment]
I have another PR that encompasses this PR

**Pull request closed without merging**